src/main/java/com/skhu/luxuryshop/user/controllerAdvice/UserExceptionControllerAdvice.java

```diff
@@ -0,0 +1,51 @@
+package com.skhu.luxuryshop.user.controllerAdvice;
+
+import com.skhu.luxuryshop.user.exception.DuplicatedEmailException;
+import com.skhu.luxuryshop.user.exception.FailedToSignupException;
+import com.skhu.luxuryshop.user.exception.SignupPasswordUnmatchedException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+@Slf4j
+public class UserExceptionControllerAdvice {
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public ResponseEntity<String> methodArgumentNotValidException(MethodArgumentNotValidException e) {
+        log.error(e.getMessage(), e);
+        return new ResponseEntity("유효성 검증 실패", HttpStatus.BAD_REQUEST);
```

Comment:
어떤 내용의 유효성 검증에 실패했는지 정확히 알려주어야 좋습니다!
프론트는 서버에서 온 이 에러 메세지를 받아서 사용자에게 보여줘야하는데
사용자 입장에서는 `유효성 검증 실패`만 딱 보이면 뭘 잘못 입력했는지 알 수가 없기때문입니다.
이 부분 뿐만 아니라 이렇게 에러 메세지를 보낼때는
 프론트에서 처리를 잘할 수 있고, 사용자가 쉽게 알아들을 수 있게 작성해주어야 좋습니다!
validation으로 검증할때
```java 
@Email(message = "이메일 형식이 맞지 않습니다.")
```
이런식으로 메세지를 구체적으로 보내고
```suggestion
    return ResponseEntity.badRequest().body(e.getMessage());
```
이렇게 처리해주면 될 것 같습니당.